### DISCUSSION TITLE
Alwrity, Give me Feedbacks

### DISCUSSION THREAD

**Discussion by AJaySi** [ALwrity/ALwrity#13] [Q&A] [answered]
1). How is Alwrity's content quality ?
2). Is alwrity, factually correct ?
3). Are your blogs ranking in search results ?
4). Suggest one improvement for alwrity ?

**Comment by KlotzJesse** [comment]
Tried to convert the output to german. Got only result of 25 lines of words.

Could you make it more / easily tweakable in the config and also prioritize maybe other language support, e.g. german? :)
All in all - looks great and could be the next big thing :) @AJaySi 

Also an major feature would be longtext -> about 1000-3000 words :)

**Reply by AJaySi** [reply]
@KlotzJesse @umesh070 @DikshaDisciplines 

I have included the following features:
1). Multilingual : Please confirm new changes.
2). Long form content, more than 3000+ words.
3). Making search engine more config driven and multilingual(WIP).

**Comment by AJaySi** [comment]
Thank you so much, Jesse. Means a lot and gives motivation to continue working on this.

1). I will update the code soon and make it multilingual.
2). Yes, for my testing, I have capped the output to around 1000 words. I will improve on it.
Note: I am traveling, so forgive delays.

Regards.

**Comment by uniqueumesh** [comment]
Content generated by Alwrity has demonstrated outstanding performance on the Search Engine Results Pages (SERP). Over the 3 weeks, I have published a total of 190 blog articles produced by Alwrity. Consequently, 150 blogs are indexed and now receiving impressions, with 40 of them beginning to attract clicks and user engagement.

**Reply by AJaySi** [reply]
Thanks a lot Umesh.. This means a lot to me. 
I am making more changes and using chatgpt4 will give better results.

Regards.

**Comment by AJaySi** [comment]
@umesh070  This is where we keep the feedback. Thanks.

https://github.com/AJaySi/AI-Writer/discussions/60

**Comment by uniqueumesh** [comment, marked as answer]
After implementing ALwrity content from February 2024 on our adventure domain website, DreamAdventures (https://www.dreamadventures.in/), we observed a significant increase in traffic. I’ve attached two Google Search Console reports to highlight the changes:

From July 1, 2023, to February 1, 2024 (before ALwrity):

Total Clicks: 14.3K
Total Impressions: 697K
Average CTR: 2.1%
Average Position: 13

![Screenshot (155)](https://github.com/user-attachments/assets/241b2b9a-5be4-4d64-adb0-e09991b74f67)

From February 1, 2024, to October 1, 2024 (after ALwrity content):

Total Clicks: 67.6K (nearly 5x increase)
Total Impressions: 3.07M (over 4x increase)
Average CTR: 2.2%
Average Position: 10.7

![Screenshot (156)](https://github.com/user-attachments/assets/4ac575b2-e616-4a92-96a3-33dd4675d115)


The significant improvement in impressions and clicks clearly shows ALwrity content's positive impact on our SEO performance. The traffic surge helped our website rank higher and attract more visitors in a highly competitive market.

Thank you, @AJaySi and ALwrity, for helping us achieve these results!

**Reply by AJaySi** [reply]
@DikshaDisciplines @uniqueumesh 

Thank you so much Umesh. This will help motivate us further. Looking forward to your contributions.
Mast.

**Comment by AJaySi** [comment]
**Hey liam,**

### On Bulk generation:

Dont do it. You will eventually get penalised/profiled by google. Take pride in your content, bulk generation compromises quality. AI is an assistant not a replacement. Every second person wants ALwrity to generate bulk content, but we will not support it. We want every piece of content audited by human before publishing. ALwrity, has all the tools to do bulk generation, you can take those pieces and do bulk generation. But, I will advise against it.

I could not access your script, maybe its private: https://github.com/liamke8/SEO/blob/main/Gomini/Gemini11.py
You are a programmer, you created a tool that some companies charge $20 a month for. Congratulations.

_**Currently, I'm not able to use Alwrity for content creation because of these issues.**_

Really sorry. Me/ @uniqueumesh have been working hard on revamping alwrity and its broken now. Request your patience for 10 days more.  @uniqueumesh has become a zombie over this. @DikshaDisciplines disciplines us on quality aspect of it.

**_Could your tool potentially support this type of bulk content creation?_**

Trust me, bulk generated content will surely get penalised by search engines. We have enhancement tickets for internal and external linking and this should be done in 2 weeks timeframe. Alwrity, already has implementation but its not tested.

SemRush and ubersuggest:

We want to make all our own FOSS tools, so ALwrity users dont have to pay for keyword research and other APIs. We are creating our own solutions for platforms that require payment. For content publishing, wordpress, wix integration code is already present and now we are working on social media publishing and AI analytics.
 
P.S : Couldnt access your codebase. Request you to collaborate on ALwrity, we need to come together to compete against paid alternatives. **ALwrity has been a failure, as its difficult to install for content creators and digital marketers.** We need to come together against the ([online Bias](https://www.alwrity.com/post/online-marketing-is-biased) and [online cost](https://www.alwrity.com/post/cost-of-online-presence-for-solopreneurs).

Thank you Liam and let me know if you want to contribute to ALwrity, I will hand hold you on new features Or show you how to fix issues you have reported. Regards.

**Comment by uniqueumesh** [comment]
#  Feedback on AI YouTube Thumbnail Generator Tool

I tested the YouTube Thumbnail Generator module in ALwrity and found the overall experience to be smooth, intuitive, and creatively empowering. Here’s a breakdown of the specific strengths I observed:

## 1. Structured Workflow with Clear Segmentation

The thumbnail generation flow is well-structured into logical steps:
Basic Info → Style & Generation → Edit & Download.

Each step is visually separated with clear tabs and button states (like the highlighted “Basic Info” or “Style & Generation” tab), reducing user confusion and helping maintain focus.

## 2. Thoughtful Input Fields for Content Context

The Basic Info section is thoughtfully designed. Fields like:

Video Title

Video Description

Target Audience

Content Type (dropdown with categories)
...ensure that the generated thumbnails are context-aware and audience-targeted.

This makes the results much more aligned with marketing principles and YouTube SEO.

## 3. Creative Concept Generation (with Strategy Built-In)

The Thumbnail Concepts feature is brilliant. It gives you multiple (1 - 5) suggestions of thumbnail in text. I can choose any concepts and generate it according to my needs.

For example, “Emotion/Hook,” “Color,” and “Why Effective” for each concept.

All the concepts particularly stood out with a detailed visual description (hand pointing, 10 numbered icons, urgency, and bright color usage). This is gold for creators who want to quickly choose the most click-worthy design.

## 4. Editable Instructions for Personalization

The Edit Thumbnail section allows users to enter natural language instructions like:

"Make background darker, add a red border, change text color to white."

This NLP-based design editing approach drastically lowers the barrier for non-designers, making AI edits extremely accessible.

## 5. Download and Final Export Feature

The “Download Thumbnail” button is placed exactly where it’s needed — at the final step after review — keeping the workflow intuitive and outcome-driven.

## 6. Polished UI Elements and Consistent Styling
Buttons, tabs, and input areas are consistently styled, with good contrast and spacing.

The use of color for active states (e.g., selected tab underlines in red and blue buttons) offers clear feedback to the user.

***

Overall, this is a thoughtfully built and highly functional thumbnail generator. Major kudos to the design and @AJaySi  — this feature will save hours for content creators and improve click-through rates significantly. 🔥👏

**Reply by AJaySi** [reply]
Thank you so much Umesh. YOU MADE MY DAY
I am really happy that you liked it, most of time you only complain. :)
@uniqueumesh
Thank you so much for your great work and making ALwrity keep going. If it wasnt for you, ALwrity would be dead long back. Thanks a lot.
